docs/sources/configure-client/language-sdks/dotnet.md

```diff
@@ -9,24 +9,53 @@ aliases:
 
 # .NET
 
-## Supported platforms
+Our .NET Profiler is a powerful tool designed to enhance the performance analysis and optimization of .NET applications. It seamlessly integrates with Pyroscope, offering real-time insights into the resource usage and bottlenecks within your .NET codebase. This integration empowers developers to pinpoint inefficiencies, improve application speed, and ensure resource-efficient operation.
 
-| Spy Name     | Type         | Linux | macOS | Windows | Docker |
-|:------------:|:------------:|:-----:|:-----:|:-------:|:------:|
-| dotnetspy    | `embedded`   |   ✅  |       |         |        |
+### Supported Profiling Types
 
-## Supported .NET versions:
- - .NET 6.0
- - .NET 7.0
+The .NET Profiler supports the following profiling types:
 
-## How to add .NET profiling to your application
+* CPU
+* Wall time
+* Allocations
+* Lock contention
+* Exceptions
+
+### Compatibility
+
+Our .NET profiler works with the following .NET versions:
+* .NET 6
+* .NET 7
+* .NET 8
+
+
+## Before you Begin
+
+### Set Up a Pyroscope Server
+
+To capture and analyze profiling data, set up a Pyroscope server. This can be:
+
+* A **local server** for development, or
+* A **remote server** for production use.
+
+For installation instructions, see our [Get Started]({{< relref "../../get-started" >}}) guide.
+
+
```

Comment:
The before you begin section details the prerequisites, which you have. It doesn't seem like you need the heading for set up a Pyroscope server since that's stated in your first sentence. 

I'm curious about the mention of Cloud Profiles. Is the real prereq that a user needs to have a profiling data storage and they can use either server or cloud to do that? If so, we might want to add a sentence immediately after the heading along the lines of: "To capture and analyze profiling data, you need either a Pyroscope server or Grafana Cloud Profiles(requires a free Grafana Cloud account.) And then go into additional details. 

```suggestion
## Before you Begin

To capture and analyze profiling data, you need either a Pyroscope server or Grafana Cloud Profiles(requires a free Grafana Cloud account.) 

### Pyroscope server

The Pyroscope server can be:

* A **local server** for development, or
* A **remote server** for production use.

For installation instructions, see our [Get Started]({{< relref "../../get-started" >}}) guide.
```